scripts/deploy-commands.js

```diff
@@ -24,32 +22,40 @@ const { REST } = require("@discordjs/rest");
 const { Routes } = require("discord-api-types/v9");
 
 const { getOrLoadAllCommands } = require("../dist/handlers/commandHandler");
-const { getApiToken, getGuildId, getClientId } = require("../dist/helpers/configHelpers");
+const {
+	getApiToken,
+	getGuildId,
+	getClientId,
+} = require("../dist/helpers/configHelpers");
 
 const API_TOKEN = getApiToken();
 const CLIENT_ID = getClientId();
 const GUILD_ID = getGuildId();
 
-const isGlobal = process.argv.some(x => x === "--global");
-const isUndeploy = process.argv.some(x => x === "--undeploy");
+const isGlobal = process.argv.some((x) => x === "--global");
+const isUndeploy = process.argv.some((x) => x === "--undeploy");
 
 if (!API_TOKEN || !CLIENT_ID) {
 	console.log("Aborting command deployment");
 	console.log("DISCORD_API_TOKEN or CLIENT_ID missing from the .env file.\n");
-	return;
+	process.exit(1);
 }
 
 if (isUndeploy && !GUILD_ID) {
 	console.log("Aborting undeployment of guild commands");
-	console.log("GUILD_ID is missing from the .env file, assuming no guild commands need to be undeployed.\n");
-	return;
+	console.log(
+		"GUILD_ID is missing from the .env file, assuming no guild commands need to be undeployed.\n"
+	);
+	process.exit(1);
 }
 
 if (!isGlobal && !GUILD_ID) {
 	console.log("Aborting guild command deployment");
 	console.log("GUILD_ID is missing from the .env file.");
-	console.log("Hint: If you just want to start the bot without developing new commands, type \"npm start\" instead\n");
-	return;
+	console.log(
+		'Hint: If you just want to start the bot without developing new commands, type "npm start" instead\n'
+	);
```

Comment:
```suggestion
	console.log('Hint: If you just want to start the bot without developing new commands, type "npm start" instead\n');
```

There are multiple examples like this, it looks off. I guess prettier is not pleased with the column count, but this looks a lot worse than before, especially when the lines right above are not new-lined like this

Reply:
I could increase the maximum line length, if you'd like - it's currently set to `80`, some projects use `120` instead.

Reply:
@nchristopher I would prefer at least 120, yeah!